style_guides/js_style_guide.md

```diff
@@ -344,92 +476,69 @@ if (a == '') {
 
 ## Only use ternary operators for small, simple code
 
-And **never** use multiple ternaries together
-
-*Right:*
+And *never* use multiple ternaries together
 
 ```js
-var foo = (a === b) ? 1 : 2;
-```
-
-*Wrong:*
+// good
+const foo = (a === b) ? 1 : 2;
```

Comment:
The good/bad examples aren't really intended to universally demonstrate alternative implementations of the same code.  In some cases, like this one, we're just showing that some circumstances are appropriate for ternary operators while some are not.